### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -311,6 +311,41 @@ ___TEMPLATE_PARAMETERS___
...
+      {
+        "type": "GROUP",
+        "name": "commonGroupPropertyRules",
+        "displayName": "Group Property Rules",
+        "groupStyle": "ZIPPY_CLOSED",
+        "subParams": [
+          {
+            "type": "SIMPLE_TABLE",
+            "name": "groupMappingRules",
+            "displayName": "Group Property Mapping Rules",
```
Comment (jethron) [inline template.tpl]: Technically it's the "_Groups_ Property Mapping Rules". :neckbeard:

File: template.tpl [hunk 2]
```diff
@@ -311,6 +311,41 @@ ___TEMPLATE_PARAMETERS___
...
+              },
+              {
+                "defaultValue": "",
+                "displayName": "Amplitude Mapped Key (optional)",
+                "name": "mappedKey",
+                "type": "TEXT",
+                "isUnique": false
+              }
+            ],
+            "help": "Specify the Property Key from the GTM Event, and then key you could like to map it to or leave the mapped key blank to keep the same name. You can use Key Path notation here (e.g. `x-sp-tp2.p` for a Snowplow events platform or `x-sp-contexts.com_snowplowanalytics_snowplow_web_page_1.0.id` for a Snowplow events page view id (in array index 0). These keys will populate the Amplitude `groups` object."
```
Comment (jethron) [inline template.tpl]: s/then key you could/the key you would/

Comment (jethron) [reply template.tpl]: Should we mention that this requires the Accounts add-on so people don't accidentally populate it? Sounds like it's a paid extra feature so not everyone will have it. Seems like we only mention it in a code comment at the moment.

File: metadata.yaml
Comment (jethron) [inline metadata.yaml:1]: Doesn't matter for this PR, but we should consider switching the CI to something like `yamllint -d "{extends: relaxed, rules: {line-length: {max: 120}}}" metadata.yaml` (per example at https://yamllint.readthedocs.io/en/stable/configuration.html#custom-configuration-without-a-config-file)

**Review by jethron — APPROVED** [review]
Mostly LGTM!

**Review by igneel64 — APPROVED** [review]

**Pull request merged**